databases/week2/part1.sql

```diff
@@ -0,0 +1,60 @@
+/*  Part 1: Working with tasks
+ Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+Change the title of a task
+Change a task due date
+Change a task status
+Mark a task as complete
+Delete a task
+ */
+USE DATABASE `hyfmodels`;
+
+select *
+from user
+where name like 'Anamika%';
+
+select *
+from status;
+
+select *
+from task;
+
+select * from user_task where user_id = 12;
+select * from user_task where task_id = 1;
+
+/*  Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id */
+insert into task (title, description, created, updated, due_date, status_id)
+values ('read a book', 'Maybe something about parenting?', '2023-01-11 06:14:01', '2023-01-11 06:14',
+        '2023-02-11 06:14:01', 2);
```

Comment:
You forgot to insert user_id here as per the Q. Better to test before post
```
insert into task (title, description, created, updated, due_date, status_id, user_id)
values ('read a book', 'Maybe something about parenting?', '2023-01-11 06:14:01', '2023-01-11 06:14', '2023-02-11 06:14:01', 1, 2);
```

Reply:
I have auto increment set to the id column for the task table, that's why Id is not set manually

Reply:
create table task
(
    id          int unsigned auto_increment
        primary key,
    title       varchar(255) not null,
    description text         null,
    created     datetime     not null,
    updated     datetime     not null,
    due_date    datetime     null,
    status_id   int unsigned not null,
    user_id     int unsigned null,
    constraint fk_status
        foreign key (status_id) references status (id)
            on delete cascade,
    constraint fk_user
        foreign key (user_id) references user (id)
            on delete cascade
)
    collate = utf8mb4_unicode_ci;